.local/bin/workman

```diff
@@ -152,6 +165,7 @@ new() {
 
     echo "${YELLOW}::${RESET} Adding ${UPSTREAM_NAME} remote at \"${UPSTREAM_URL}\"..."
     git remote add "${UPSTREAM_NAME}" "${UPSTREAM_URL}"
+    git config --add remote."${UPSTREAM_NAME}".fetch '+refs/pull/*/head:refs/remotes/${UPSTREAM_NAME}/pr/*'
```

Comment:
I think this would be better behind a `FETCH_UPSTREAM_PRS=1` option. 

Personally, I [alias](https://github.com/davidtwco/dotfiles/blob/440628ee9d73af779a9b4ae53f18fbafb8611311/.aliases#L67-L70) [`hub`](https://github.com/github/hub) to `git` and then run `git pr checkout 12345` when I want to check out a PR.

Reply:
TIL about `hub`